trussworks/react-uswds
# fix: Fix Tooltip positioning

# Summary

Updates the code used to position the Tooltip component to reflect [USWDS 2.11.2 Tooltip source code](https://github.com/uswds/uswds/blob/v2.11.2/src/js/components/tooltip.js). This seems to resolve the positioning bug.

## Related Issues or PRs

Fixes #1877 

## How To Test

View the Tooltip component in Storybook at various screen sizes and verify the tooltip is always visible and positioned as expected.

### Screenshots (optional)

![image](https://user-images.githubusercontent.com/2723066/149024245-30f28f87-fbc4-45ae-85c6-12e7c11c316d.png)

![image](https://user-images.githubusercontent.com/2723066/149024275-41721aae-3d2d-4172-b5cd-95b8d14096ed.png)
